### ISSUE TITLE
Feature: detect and parse http links and make them clickable

### ISSUE THREAD

**Issue by onefuncman** [onefuncman/telnet-proxy#3]
As part of the conversation tracking feature (depends on #4 ), look for links, attribute them to their source, and make them clickable.  Stretch goal to add a on-hover preview (maybe when a modifier key is held while hovering the link, like control or shift)
